src/options/options_handler.cpp

```diff
@@ -1086,7 +1092,8 @@ theory::bv::SatSolverMode OptionsHandler::stringToSatSolver(std::string option,
 
     if (options::bitblastMode() == theory::bv::BITBLAST_MODE_LAZY &&
         options::bitblastMode.wasSetByUser()) {
-      throw OptionException(std::string("Cryptominsat does not support lazy bit-blsating. \n\
+      throw OptionException(
+          std::string("Cryptominsat does not support lazy bit-blasting. \n\
```

Comment:
CryptoMiniSat or at least Cryptominisat (instead of Cryptominsat)

Reply:
Done.